Suppress more warnings

tugsbayasgalan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #149833



cc @ezyang @SherlockNoMad @EikanWang @jgong5 @wenzhe-nrv

Differential Revision: [D71702307](https://our.internmc.facebook.com/intern/diff/D71702307)

tugsbayasgalan committed:
Suppress more warnings

[ghstack-poisoned]

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D71702307).

tugsbayasgalan commented:
@pytorchbot rebase

tugsbayasgalan committed:
Update

[ghstack-poisoned]

Skylion007 commented on torch/_export/passes/lift_constants_pass.py:
```diff
@@ -213,9 +216,11 @@ def lift_constants_pass(
+                    log.debug(
+                        "%s created when tracing %s is a parameter. But "
+                        "it's not registered with register_parameter(). export will treat it as a constant tensor",
+                        str(node.target),
+                        str(node.meta.get("stack_trace", "<unknown stack>")),
```
Str will be called implicitly by the % formatting. No need to call it here.

Skylion007 replied:
```suggestion
                        node.target,
                        node.meta.get("stack_trace", "<unknown stack>"),
```

tugsbayasgalan committed:
Update on "Suppress more warnings"




cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

Differential Revision: [D71702307](https://our.internmc.facebook.com/intern/diff/D71702307)

[ghstack-poisoned]

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D71702307).
